storage/client/src/main/java/com/azure/storage/blob/models/BlockBlobItem.java

```diff
@@ -0,0 +1,69 @@
+package com.azure.storage.blob.models;
+
+import com.azure.core.implementation.util.ImplUtils;
+
+import java.time.OffsetDateTime;
+
+public class BlockBlobItem {
```

Comment:
This appears to be just the class AppendBlobItem. It contains fields specific to append blobs.

Reply:
Unrelated items removed.